### PR TITLE
Notebooks update

### PR DESCRIPTION
**Pull request by jburel**
* Remove dependencies
* Fix URL

To test:
* Check that the python notebooks are passing

Accessible at https://ome-lochy.openmicroscopy.org/jupyterhub-training

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
...
- A set of Notebooks to demonstrate how to access the images and metadata from OMERO
+
+ A set of Notebooks to demonstrate how to access the images and metadata from OMERO.
+
+ The notebooks in this repository are meant to exemplify how to access data in OMERO.
+
+ | **Notebook** | **Lang** | **Description** |
+ | **[OMERO example](Python/OMEROHelloWolrdNotebook.ipynb)** | Py | Shows how to connect to OMERO and read data. |
+ | **[Calculate Sharpness](Python/CalculateSharpness.ipynb)** | Py | Calculates sharpness of images and generates heatmaps. |
+ | **[Illumination Correction](Python/CalculateSharpness.ipynb)** | Py | Correct the selected image. |
```
Comment (manics) [inline README.md]: The table isn't formatted correctly:
screen shot 2018-05-07 at 10 01 12

Try http://www.tablesgenerator.com/markdown_tables ?

Comment (jburel) [reply README.md]: Need to go back to that

**Comment by manics** [comment]
Looks fine. `pixels.getPlane()` is failing for me but that should be unrelated to this PR.

**Comment by jburel** [comment]
Which server did you use?
Did you change the image_id?

**Comment by manics** [comment]
outreach user-1. The image loaded but getPlane failed.

**Comment by jburel** [comment]
Something to do with the reader
``
serverStackTrace = ome.conditions.InternalException:  Wrapped Exception: (java.lang.NullPointerException):
null
	at loci.formats.in.DicomReader.openBytes(DicomReader.java:292)
``
this will require investigation
not related to this PR

**Pull request merged**